src/utils/safelyInvoke.spec.js

```diff
@@ -0,0 +1,17 @@
+import safelyInvoke from './safelyInvoke';
+
+describe('safelyInvoke', () => {
+  test('should return the original function', () => {
+    const myFunction = jest.fn();
+    expect(safelyInvoke(myFunction)).toBe(myFunction);
+  });
+
+  test('should return an empty function if called with non-function parameter', () => {
+    const nonFunction = undefined;
+    expect(typeof safelyInvoke(nonFunction)).toBe('function');
+  });
+
+  test('should not throw errors when the provided empty function is called', () => {
+    safelyInvoke(undefined)();
```

Comment:
Actually, jest has a matcher for asserting that a function doesn't throw an error. So something like this.

```
expect(() => {
  safelyInvoke(undefined)();
}).not.toThrow();
```

Reply:
Sure, actually we can omit the function wrapper too as `safelyInvoke()` will return a function and we can pass that to `expect()` as-is.

Reply:
The reason why I put the function wrapper is that we need to call the function that safely invoke returns. If we don’t it is the same as doing 

expect(2).not.toThrow()